EXPORT_HELPERS.md

```diff
@@ -0,0 +1,215 @@
+## Proposal / Discussion Draft: Consistent & Documented Export Formats
+
+Goal of this proposal is to produce consistent and conformant formatted exports of the VSS via
+the VSS-TOOLS or other potential 3rd-party implementations, resulting in a high degree of stability and
+interoperability between implementations using a particular format (such as IDL).
+
+This consistency and stability can ease the creation of interoperable examples and reference implementations for a given format.  
+This draft focuses on IDL, but is intended to also maximize interoperability within other export formats.
+
+To enable this capability, the following changes are proposed:
+
+- Create a document to describe the recommended formatting and structure of the resulting exported (IDL or other) files.
+- Modify the exporters in VSS-TOOLS to conform to the above document.
+- Utilize the `arraysize` and `stringsizemax` to set the dimensions of the exported array/sequence/string types.
+- Introduce 3 new keywords to the VSS: `memberof`, `membergroup`, `isogroup`, used as:
+    - To have a `branch` entity create a namespace in the export (named for itself), add:
+        - `memberof: namespace`
+    - To create named group containers in the export for elements, add to the branch entry:
+        - `membergroup <list of names>` to create containers that may also be referenced in the parent of this branch.
+        - `isogroup <list of names>` to create containers that will not be referenced in the parent of this branch.
+    - To assign `membergroup` struct names to be referenced in a parent struct:
+        - `memberof: <list of parent struct names to insert reference into>`
+    - To assign VSS elements to specific named group containers in the export, add to element entry:
+        - `memberof: <list of group containers>`
+
+## Effect on IDL Export
+Implementing the above keywords into the VSS -- to enforce the guidance of the above reference document -- 
+should have no effect on existing exporters in VSS-TOOLS, however a modified IDL exporter could use this new
+information to consistently produce IDL that conforms to the above document.
+
+Examples of this export guidance on IDL includes:  
+
+### Namespaces
+In IDL, namespaces are expressed with a `module { ... }` container around the affected definitions.

```

Comment:
Just a nitpick, but as I assume this refers to DDS/OMG IDL, maybe when referring to it, should be spelt out, because in my view "IDL" is a pretty generic term, that is applicable to many different IDLs?